pyvisa-py/protocols/usbtmc.py

```diff
@@ -127,6 +130,7 @@ def from_bytes(cls, data):
     def from_quirky(cls, data):
         """Constructs a correct response for quirky devices"""
         msgid, btag, btaginverse = struct.unpack_from('BBBx', data)
+        data = data.rstri(b'\x00')
```

Comment:
there is a typo here. Please test the code I do not have a device to test against.

Reply:
The original code from https://github.com/pyvisa/pyvisa-py/issues/20 was tested against a Siglent SDG 805. I have access to a RIGOL DG 1062Z and a RIGOL 1052E. I am currently involved, and will try find the time to test the pull request in the second week of may.